src/main/java/org/jenkinsci/plugins/gitclient/JGitAPIImpl.java

```diff
@@ -47,6 +47,7 @@
 
 import javax.annotation.Nullable;
 
+import jenkins.model.Jenkins;
```

Comment:
stephenc: > I am concerned greatly that you are importing Jenkins as that is a class that *should* (never mind that other plugins such as Subversion are bold and do bad things, but we should be getting better not worse) not be loading on agents. Given that we explicitly construct this class on a remote agent I would expect to see sufficient classloading guard methods to ensure that the methods referencing `Jenkins` are never called on execution paths which will be invoked on the agent and are only for execution paths running on the `Jenkins` master JVM... that is almost impossible to do without specifying a core dependency of 1.653+ (see `JenkinsJVM` class added in 1.653) so I think this is a bad change